psiturk/amt_services.py

```diff
@@ -26,13 +23,13 @@ class AmtServicesResponse(object):
     """class AmtServicesResponse."""
 
     def __init__(self, status=None, success=None, operation='', message='',
-                 data={}, **kwargs):
```

Comment:
good! looks like this was another mutable default arg bug that might have bit hard later